test/core/templates/options-test.js

```diff
@@ -0,0 +1,59 @@
+const test = require('tape'),
+  base64Img = require('base64-img'),
+  looksSame = require('looks-same');
+
+const ImageSequencer = require('../../../src/ImageSequencer');
+
+const red =
+  'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQAQMAAAAlPW0iAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABlBMVEX+AAD///+KQee0AAAAAWJLR0QB/wIt3gAAAAd0SU1FB+EGHRIVAvrm6EMAAAAMSURBVAjXY2AgDQAAADAAAceqhY4AAAAldEVYdGRhdGU6Y3JlYXRlADIwMTctMDYtMjlUMTg6MjE6MDIrMDI6MDDGD83DAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDE3LTA2LTI5VDE4OjIxOjAyKzAyOjAwt1J1fwAAAABJRU5ErkJggg==';
+target = 'test_outputs';
+
+/**
+ * @method optionsChangeTest
+ * @description a common test for modules
+ * @param {String} moduleName name of the module
+ * @param {'Object'} [options] array of options
+ * @param {String} [benchmark] dataURI of the benchmark images, a image for easch option
+ * @param {String} [input='red_image'] optional input image. Default is a red image.
+ */
+module.exports = (moduleName, options, benchmark, input) => {
+  let sequencer = ImageSequencer({ ui: false });
+
+  test(`${moduleName} module works correctly with different options`, t => {
+    sequencer.loadImages(input || red);
```

Comment:
Could you add some more inline comments describing the code below, to make it easy to understand for future readers.

Reply:
Done!